packages/terra-core-docs/src/terra-dev-site/doc/action-header/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,251 @@
+  - For any actionable element, avoid non-unique labels unless the element perform the exact same function. For example, two print buttons that print the exact same document are fine. However, if the buttons print different documents, this would be an accessibility failure. 
+    - If non-unique names cannot be avoided, inform the engineer of other text on the page that can be programmatically associated with other content on the page that may help describe a visual relationship. For example, multiple “Learn more” buttons on the page are placed in visual proximity to the item the user can learn more about. Ensure the first word(s) of the programmatic label match the visual label. 
+  - When using icons, inform engineers when an icon is purely decorative. Otherwise, provide the engineer with meaningful alternate text which will serve as the accessible name.  
+    - Ensure any icon used within Action Header is used to consistently represent the same functionality. Additionally, any alternate content used with an icon must be consistent for a predictable user experience. The alternate text should be consistent in wording based on the icon usage 
+- Ensure the engineers understand the logical reading order of the page. 
+- Never add unexpected behaviors to buttons that could cause a change of context on focus or activation of the button. Ensure buttons always meet the user’s expectations.  
+- After coded, ensure all interactable elements receive keyboard focus in the user’s reading order and display a visible keyboard focus. Ensure the keyboard user does not get “trapped” somewhere on the page preventing them from the entire workflow. 
+
+## Usability Expectations:
+Users do not expect Action Header to be actionable. They would, however, expect any actionable elements within Action Header to perform the expected action based on the label of the element. 
+
+### Interaction Details
+Users do not expect Action Header to receive keyboard focus or be actionable. However, they would expect any actionable element within it to receive keyboard focus and behave in an expected and predictable manner. 
+<Whitespace />
+Mouse users expect the cursor to change to a hand with pointing finger when a Button or Link is hovered. Keyboard only users expect to see a visible keyboard focus indicator when an actionable element is in focus. 
+
+### Keyboard Navigation
+
+The basic keyboard navigation expectations: 
+
+| Key/Sequence | Description |
+|---|---|
+|*Tab*| Moves focus on and off interactable elements. User expects elements to receive focus in the logical reading order of the page.  |
+|*Enter*| Executes Button action.   |
+
+<Whitespace newline={4} />
+
+## Support Compliance:
+
+### Primary criteria related to Terra Action Headers 
+- [1.3.1 Info and Relationships](https://wiki.cerner.com/display/UserExperience/1.3.1+Info+and+Relationships) — Supports 
```

Comment:
Instead of giving the internal wiki.cerner links it is better to give w3.org links for all the support compliance criteria.